### ISSUE TITLE
Dashboard metrics filter popup extends beyond browser window and disappears when I try to scroll the page

### ISSUE THREAD

**Issue by KalleOlaviNiemitalo** [dotnet/aspire#3137]
When I click a filter button on the "Metrics" page of the Aspire dashboard, it pops up a checkbox list of known values of the attribute.  Depending on the size of the Web browser window and the current scrolling position, it is possible that this checkbox list popup is not fully visible, as part of it extends beyond the bottom of the browser window.  The Web browser shows a scroll bar at the edge of the window (and perhaps another scroll bar at the edge of the checkbox list popup), but when I try to scroll the page by dragging the scroll bar at the edge of the window, the scroll box does not move; and when I release the mouse button on the scroll bar, the checkbox list popup disappears.

Reproduced on Microsoft Edge 123.0.2420.53.

**Comment by davidfowl** [comment]
Picture or video?

**Comment by adamint** [comment]
I'm not able to reproduce this currently @KalleOlaviNiemitalo. However, @vnbaaij I did notice that FluentPopover is not respecting the `AutoFocus="false"` parameter set on the filter popover in this case (unable to scroll using mouse scroll), which could be causing this issue. Just want to confirm that the behavior I am seeing is an unintended issue with FluentPopover

**Comment by vnbaaij** [comment]
That is not what the Autofocus parameter does. It is intended to automatically place the focus on the popover so keyboard interaction can take place (when set to true). 
A popover uses an Overlay by default and that can indeed makes the background inert.

**Comment by adamint** [comment]
@vnbaaij is there a way to set the Overlay `PreventScroll` parameter on the popover, then?

**Comment by vnbaaij** [comment]
No, there isn't. The `FluentOverlay` is internal to the `FluentPopover`.

But I don't think this issue is relevant anymore. I also can't reproduce it. The Filter popover dispalys in the correct location relative to the button basedon the available screen estate (ie button is at bottom of the window->Popover displays above instead of under the button)
![metrics-filter](https://github.com/dotnet/aspire/assets/1761079/6eecee35-518b-43fe-b023-11e5a09fdf23)

**Comment by danmoseley** [comment]
@KalleOlaviNiemitalo (Just picked this issue at random) -- I have a question to ask you. Are you comfortable sending me a mail at danmose@m....com ? LMK if you'd rather some other way.

**Comment by danmoseley** [comment]
> @KalleOlaviNiemitalo (Just picked this issue at random) -- I have a question to ask you. Are you comfortable sending me a mail at danmose@m....com ? LMK if you'd rather some other way.

I will be out on leave shortly so it would be great @KalleOlaviNiemitalo if you could add joperezr@...  to the mail as well.

**Comment by joperezr** [comment]
As part of triage cleanup ahead of the 13.3 release, this issue is being moved to the Backlog milestone. If you believe it should remain in 13.3, feel free to re-add to the milestone and leave a comment here why it should not be punted.

**Comment by adamint** [comment]
This is actually a larger issue. Drop downs for commands, for example, on the resources page can also extend past the page, which I believe has the same root cause.
